### PR TITLE
Implement table with mocked data

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

**Comment by farebord** [comment]
Is there a reason we use `moment` instead of the default [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object?

**Comment by VargaElod23** [comment]
Out of personal convenience, honestly. I've been working with `moment` for a long time now and my personal experience is that developers that manipulate the code later on do it so much easier because moment's syntax is much more readable that  native Date object manipulation...

File: services/community-site/src/pages/Redeem/redeem.scss [hunk 1]
```diff
@@ -1,3 +1,15 @@
+.container {
```
Comment (rattrap) [inline services/community-site/src/pages/Redeem/redeem.scss:1]: You need to add these in a `.redeem` wrapper or something. Or else all of these classes will be applied to everywhere.

File: services/community-site/src/utils/claim.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { ethers } from 'ethers';
```
Comment (rattrap) [inline services/community-site/src/utils/claim.ts:1]: I wouldn't add it to utils. Maybe create a new service for this?

**Review by rattrap — CHANGES_REQUESTED** [review]

File: services/community-site/src/pages/Redeem/redeem.scss (hunk 1 above)
Comment (Elod23) [reply services/community-site/src/pages/Redeem/redeem.scss:1]: I'm sorry but I don't really get what you mean by this. Aren't these classes only on the scope of the JSX file it gets imported into?

File: services/community-site/src/utils/claim.ts (hunk 2 above)
Comment (VargaElod23) [reply services/community-site/src/utils/claim.ts]: Done 👍🏼

**Pull request merged**